src/js/components/LoginForm.js

```diff
@@ -21,12 +21,19 @@ export default class LoginForm extends Component {
     this.refs.username.focus();
   }
 
-  _onSubmit  (event) {
+  _onSubmit (event) {
     event.preventDefault();
-    var username = this.refs.username.value.trim();
-    var password = this.refs.password.value.trim();
+
+    let username = this.refs.username.value.trim();
+    let password = this.refs.password.value.trim();
+    let rememberMe;
+
+    if ('rememberMe' in this.refs) {
```

Comment:
just out of curiosity, do you why we have to check for rememberMe presence in refs?